tests/integrations/django/asgi/test_asgi.py

```diff
@@ -5,11 +5,19 @@
 from sentry_sdk.integrations.django import DjangoIntegration
 from tests.integrations.django.myapp.asgi import channels_application
 
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
+
 APPS = [channels_application]
+ASGI_APP = []
+
 if django.VERSION >= (3, 0):
     from tests.integrations.django.myapp.asgi import asgi_application
 
-    APPS += [asgi_application]
+    ASGI_APP = [asgi_application]
```

Comment:
You don't need this. Instead of using pytest.mark.parametrize you can just refer to `asgi_application` directly from within the test (instead of `application`). The test will only run on 3.1 anyway.